relbench/datasets/avito.py

```diff
@@ -12,6 +12,10 @@
 class AvitoDataset(RelBenchDataset):
     name = "rel-avito"
     url = "https://www.kaggle.com/competitions/avito-context-ad-clicks"
+    err_msg = (
+        "{data} not found. Please download avito data from "
```

Comment:
can you also provide the kaggle download command for convenience (same as rel-hm and rel-event), thanks!

Reply:
I could potentially provide the kaggle download command. However, the size of the Avito zip file is ~20G. See https://www.kaggle.com/competitions/avito-context-ad-clicks/data. Thus, I do not think this is very practical and it would be better if people could use the subsampled Avito dataset provided by us instead.

Reply:
That makes sense. But the current error message asks to download from the Kaggle URL. I feel it would just be more consistent with the other datasets if we provide the kaggle command here too. But approving to unblock.